reader-osm/src/main/java/com/graphhopper/reader/osm/pbf/PbfBlobDecoder.java

```diff
@@ -32,6 +34,7 @@
     private final String blobType;
     private final byte[] rawBlob;
     private final PbfBlobDecoderListener listener;
+    private final Inflater inflater = new Inflater();
```

Comment:
Is this safe here? Inflater is not thread-safe.

Reply:
Good catch!